src/controllers/authController.ts

```diff
@@ -119,3 +122,86 @@ export const sendOTP = async (req: Request, res: Response, email: string) => {
     }
   }
 };
+
+// Function to Request Password reset token
+export const forgotPassword = async (req: Request, res: Response) => {
+  try {
+    const { email } = req.body;
+
+    // Verify if user exists
+    const user = await User.findOne({ where: { email } });
+    if (!user) {
+      return res.status(404).json({ ok: false, error: 'User with this email does not exist' });
+    }
+
+    // Generate reset token
+    const token = await userToken(user.id, user.email);
+
+    // Send email with token
+    const link = `${process.env.URL_HOST}:${process.env.PORT}/api/auth/reset-password/${token}`;
+
+    await sendEmail('reset_password', {
+      name: `${user.firstName} ${user.lastName}`,
+      email: `${user.email}`,
+      link: link,
+    });
+
+    return res.status(200).json({
+      ok: true,
+      message: 'A password reset link has been sent to your email.',
+    });
+  } catch (error) {
+    logger.error('Error requesting password reset: ', error);
```

Comment:
That function already has a logger, therefore remove this line

` logger.error('Error requesting password reset: ', error);`